generated/proto/commonpb/common.proto

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
```

Comment:
I actually prefer calling it commonpb since there is a higher chance of collision with another common package, one thing I hate about the current package name in statsdex for those protos is that I always have to manually rename the import.